dbg_files.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Provie file paths to archive for debug image as relative to src subdir
+#
+SRC_DIR_LIST="\
+    libteam \
+    lldpd \
+    lm-sensors \
+    libnl3 \
+    radvd \
+    redis \
+    snmpd \
+    sonic-daemon-base \
+    sonic-dbsyncd \
+    sonic-frr \
+    sonic-platform-common \
+    sonic-platform-daemons \
```

Comment:
these are python projects, not sure if we can use gdb to debug them.

Reply:
+ few more dropped.